### ISSUE TITLE
Build logs unavailable

### ISSUE THREAD

**Issue by SylvainCorlay** [rtfd/readthedocs.org#3993]
## Details

* Read the Docs project URL: https://readthedocs.org/projects/ipyleaflet
* Build URL (if applicable): https://readthedocs.org/projects/ipyleaflet/builds/7076306/
* Read the Docs username (if applicable): SylvainCorlay

## Expected Result

I expect to see logs for the failed builds

## Actual Result

The content of the page is "latest (6938f2dbe0edc0b26c949b6d14bedead40d47772)".

**Comment by boegel** [comment]
It seems like I'm running into the same problem, see https://readthedocs.org/projects/easybuild/builds/7076258/

**Comment by stsewd** [comment]
@SylvainCorlay did you try building locally? How long (and much memory) does it take? I tried on my local instance and I had to increase the memory and time limit to build successfully (build took 1267 seconds).

**Comment by SylvainCorlay** [comment]
Yes, the docs build fine locally. The HTML build takes about half a second on my 6 years old macbook, and python takes up 46MB of memory during that time.

**Comment by SylvainCorlay** [comment]
@stsewd is the project that you tried ipyleaflet?

**Comment by SylvainCorlay** [comment]
I started another build (https://readthedocs.org/projects/ipyleaflet/builds/7076823/), and I am getting

```
Error
There was a problem with Read the Docs while building your documentation. Please report this to us with your build id (7076823).
```

**Comment by SylvainCorlay** [comment]
Starting it again yields no log at all
https://readthedocs.org/projects/ipyleaflet/builds/7076842/

**Comment by stsewd** [comment]
@SylvainCorlay yes, I tried `ipyleaflet`.

> Starting it again yields no log at all

That's because the command is still running, on my local installation, I can see that this command is `conda env create ...` which gets killed due to excessive memory consumption, so I suspect the same is happening on rtd.

See #3613

**Comment by SylvainCorlay** [comment]
I am seeing the same kind of issue (no build log) with a new project: xhale

https://readthedocs.org/projects/xhale/

The build takes one second to fail and there is no information whatsoever. 

<img width="838" alt="screen shot 2018-04-22 at 11 32 50 pm" src="https://user-images.githubusercontent.com/2397974/39100142-83a0f9f4-4685-11e8-8f10-b45c3d166cca.png">

**Comment by stsewd** [comment]
@SylvainCorlay I can see that the failure is because of a bad configuration file

![screenshot-2018-4-22 xhale read the docs](https://user-images.githubusercontent.com/4975310/39100188-e81cd9e6-464b-11e8-8d85-4c28378530b5.png)

**Comment by SylvainCorlay** [comment]
Thanks, this log appeared more than 20 hours after the build appeared as failing..

The new issue is that we don't see any log for most repositories when the build fails.

It started late last week, on multiple organizations and repositories: See e.g. the build for `pythreejs`: https://readthedocs.org/projects/pythreejs/builds/7083409/

<img width="819" alt="screen shot 2018-04-23 at 2 14 58 pm" src="https://user-images.githubusercontent.com/2397974/39125893-d93b807a-4700-11e8-9e0d-a97337a1cd75.png">

**Comment by agjohnson** [comment]
You need to refresh without cache, your browser is using an out of date JS asset. We should invalidate these assets somehow, but that's another issue.

**Comment by SylvainCorlay** [comment]
> You need to refresh without cache, your browser is using an out of date JS asset. We should invalidate these assets somehow, but that's another issue.

Thanks! This fixed the issue! I can see the logs now.

Although I am getting the infamous

```
You need to refresh without cache, your browser is using an out of date JS asset. We should invalidate these assets somehow, but that's another issue.
```

**Comment by agjohnson** [comment]
Oops yeah, that's a new problem with docker from a deploy last week. I'll close the issue here as we're addressed the docker change in #3999
